README.md

```diff
@@ -200,70 +201,65 @@ let credential = await navigator.credentials.get({
       {
         origin: "https://login.idp.net",
       },
+    ]
   }
 });
 ```
 
-or embeds an IdP iframe or other resources that want to access unpartitioned cross-site storage.
+We need to bypass the requirement for user interaction here if we had user interaction to force the navigation but redirected the current tab instead of opening a popup.
 
-This can be run on every page load as it is guaranteed to provide no browser UI and provides the cross-site unpartitioned storage access desired.
+Note that in the case of a popup, the credential chooser should show once the identity provider stores a credential that is effective for the pending credential request on the relying party, removing the need for the relying party to call `navigator.credentials.get` a second time.
 
-### Scenario 2: User intends to link to an identity provider they are already logged in to
+### Scenario 2: User logs in with one of many identity providers, or other types of credential
```

Comment:
I'm not 100% sure what Scenario 2 adds to the proposal. What is it that we want user agents to implement here?